pymongo/message.py

```diff
@@ -105,7 +105,7 @@
     "insert": "documents",
     "update": "updates",
     "delete": "deletes",
-    "bulkWrite": "bulkWrite",
+    "bulkWrite": "ops",
```

Comment:
There's a larger question here on if we should even be doing this behavior anymore at all but I'll open a new ticket to investigate that.